tools/apiview/emitters/cadl-apiview/test/test-host.ts

```diff
@@ -15,14 +15,17 @@ export async function createApiViewTestRunner({
   withVersioning,
 }: { withVersioning?: boolean } = {}) {
   const host = await createApiViewTestHost();
-  const importAndUsings = `
-    import "@cadl-lang/rest";
-    ${withVersioning ? `import "@cadl-lang/versioning"` : ""};
-    using Cadl.Rest;
-    using Cadl.Http;
-    ${withVersioning ? "using Cadl.Versioning;" : ""}
-  `;
-  return createTestWrapper(host, (code: string) => `${importAndUsings} ${code}`, {
-    emitters: { "@azure-tools/cadl-apiview": {} },
+  const autoUsings = [
+    "Cadl.Rest",
+    "Cadl.Http",
+  ]
+  if (withVersioning) {
+    autoUsings.push("Cadl.Versioning");
+  }
+  return createTestWrapper(host, {
+    autoUsings: autoUsings,
+    compilerOptions: {
+      emitters: { "@azure-tools/cadl-apiview": {} },
```

Comment:
```suggestion
      emit: ["@azure-tools/cadl-apiview"],
```

`emitters` is deprecated